src/source_map_utils.ts

```diff
@@ -135,8 +135,9 @@ export class DefaultSourceMapper implements SourceMapper {
 
   constructor(private fileName: string) {
     this.sourceMap.addMapping({
-      original: {line: 1, column: 1},
-      generated: {line: 1, column: 1},
+      // tsc's source maps 1-index lines, 0-index columns
```

Comment:
Best reference I can find is here:
https://groups.google.com/forum/#!topic/mozilla.dev.js-sourcemap/kdOU6cGsplA

@concavelenz any update or insight on that? It seems like we indeed need to use 1-indexed lines, 0-indexed columns?